devito/finite_differences/differentiable.py

```diff
@@ -369,6 +369,31 @@ def _(obj):
         return obj.__class__
 
 
+def diff2sympy(expr):
+    """
+    Translate a Differentiable expression into a SymPy expression.
+    """
+
+    def _diff2sympy(obj):
+        flag = False
+        args = []
+        for a in obj.args:
+            ax, af = _diff2sympy(a)
+            args.append(ax)
+            flag |= af
+        if isinstance(obj, DifferentiableOp):
```

Comment:
Isn't it "easier" to just have a `cls._parent`?

Reply:
it's not a direct parent due to multiple inheritance. I could add an extra field... but I wouldn't think it'd make much of a difference

Reply:
Hum ok. Was just wondering how expensive it is. I guess it's easier to add extra sympy classes like that.

Reply:
I think you're right, speed may be a concern. I'm changing this now to avoid the `isinstance`

Reply:
done